inlong-manager/manager-common/src/main/java/org/apache/inlong/manager/common/fieldtype/datasource/PostgreSQLFieldTypeMapping.java

```diff
@@ -0,0 +1,161 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.inlong.manager.common.fieldtype.datasource;
+
+import org.apache.commons.lang3.StringUtils;
+
+import java.util.stream.Stream;
+
+import static org.apache.inlong.manager.common.consts.InlongConstants.LEFT_BRACKET;
+
+/**
+ * The enum of PostgreSQL field type mapping.
+ */
+public enum PostgreSQLFieldTypeMapping implements BaseFieldTypeMapping {
```

Comment:
The enumeration class is still not easy to expand.

Is it possible to add the type mapping to the configuration file, and then the code reads and parses it?

Of course, in this issue, you can use the enumeration method first, and then you can take the time to see how to optimize it through the configuration file.

Reply:
> The enumeration class is still not easy to expand.
> 
> Is it possible to add the type mapping to the configuration file, and then the code reads and parses it?
> 
> Of course, in this issue, you can use the enumeration method first, and then you can take the time to see how to optimize it through the configuration file.

It is really better to add the configuration file, and then optimize it in the next PR